# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsFocusManager.cpp
# Commit: 8bf69c3fd814
# Full Hash: 8bf69c3fd81419337cb93f96020bb5e88a333d32
# Author: Jan Varga <Jan.Varga@gmail.com>
# Date: 2012-08-07 13:39:24
# Regressor Bug: 779684
# File Overlap Count: 1
# Description:
#   Bug 779684 - Move FrameIteratorFlags to nsFrameIterator class. r=roc
# ==============================================================================

diff -r 78aaaf0d8185 -r 8bf69c3fd814 dom/base/nsFocusManager.cpp
--- a/dom/base/nsFocusManager.cpp	Mon Aug 06 14:51:33 2012 -0600
+++ b/dom/base/nsFocusManager.cpp	Mon Aug 06 20:27:04 2012 -0400
@@ -2272,7 +2272,7 @@
             startContent != aDocument->GetRootElement()) {
           // Yes, indeed we were at the end of the last node
           nsFrameIterator frameTraversal(presContext, startFrame,
-                                         eLeaf, FrameIteratorFlags::FLAG_FOLLOW_OUT_OF_FLOW);
+                                         eLeaf, nsFrameIterator::FLAG_FOLLOW_OUT_OF_FLOW);
 
           nsIFrame *newCaretFrame = nullptr;
           nsCOMPtr<nsIContent> newCaretContent = startContent;
@@ -2698,7 +2698,7 @@
     }
 
     nsFrameIterator frameTraversal(presContext, startFrame,
-                                   ePreOrder, FrameIteratorFlags::FLAG_FOLLOW_OUT_OF_FLOW);
+                                   ePreOrder, nsFrameIterator::FLAG_FOLLOW_OUT_OF_FLOW);
 
     if (iterStartContent == aRootContent) {
       if (!aForward) {